src/Broadway/EventStore/Exception/DuplicatePlayheadException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Broadway\EventStore\Exception;
+
+use Broadway\EventStore\EventStoreException;
+use Exception;
+
+class DuplicatePlayheadException extends EventStoreException
+{
+    public static function create(Exception $exception)
```

Comment:
To be honest, I'm not even sure if you would want to assume this exception is always based on another exception. For event stores where the duplicate isn't detected based on an exception, you would need to create an exception to create an exception...

I would prefer to have a method where you pass the id and playhead that are causing the duplicate (and pass an optional previous exception), but that will be tricky as you can't really get that info from the DBALException.... But maybe we can at least report for which identifier the duplicate occurred?